### PR TITLE
.github/workflows/editorconfig.yml: separate fetch and check steps

### PR DESCRIPTION
**Pull request by zowoq**
Separate steps should be easier to read.

Also rename the tool to match what we have packaged.
```
/nix/store/bnlc49ibgdy904mlpv67h3x6jnf17xz7-editorconfig-checker-2.1.0/bin/editorconfig-checker
```

### CODE REVIEW

**Review by mweinelt — APPROVED** [review]
LGTM. Needs rebasing.

File: .github/workflows/editorconfig.yml [hunk 1]
```diff
@@ -13,12 +13,17 @@ jobs:
...
+        ECC_URL: "https://github.com/editorconfig-checker/editorconfig-checker/releases/download"
       run: |
-        curl -sSf -O -L -C - https://github.com/editorconfig-checker/editorconfig-checker/releases/download/$VERSION/ec-$OS-$ARCH.tar.gz && \
+        curl -sSf -O -L -C - $ECC_URL/$VERSION/ec-$OS-$ARCH.tar.gz && \
         tar xzf ec-$OS-$ARCH.tar.gz && \
-        ./bin/ec-$OS-$ARCH -disable-indentation ${{ env.GIT_DIFF }}
+        mv ./bin/ec-$OS-$ARCH ./bin/editorconfig-checker
+    - name: Checking EditorConfig
+      run: |
+        ./bin/editorconfig-checker -disable-indentation \
```
Comment (mweinelt) [inline .github/workflows/editorconfig.yml:28]: That line break does not feel necessary, even with the line below your below 80 characters.

Comment (zowoq) [reply .github/workflows/editorconfig.yml:28]: It's for when multiple files are edited so that they are all after the break rather than being cut off.

Comment (mweinelt) [reply .github/workflows/editorconfig.yml:28]: Okay, works for me.

**Pull request merged**